### ISSUE TITLE
Unable to declare multiple fonts as a single value

### ISSUE THREAD

**Issue by RGBboy** [css-modules/postcss-icss-values#21]
If I declare a value as a series of font names:

``` css
@value defaultFont: "Helvetica Neue", Helvetica, Arial, sans-serif;
```

and use that within a css module

``` css
.example {
  font-family: defaultFont;
}
```

I get the following output:

``` css
.example {
  font-family: "Helvetica Neue";
}
```

Ideally I would like this to output:

``` css
.example {
  font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
}
```

It would also be nice if I could compose values into a new value:

``` css
@value defaultFont: "Helvetica Neue", Helvetica, Arial, sans-serif;
@value specialFont: "MySpecialFont", defaultFont;
.example {
  font-family: specialFont;
}
```

Is this something worth implementing here? Or should I just resort to using the `composes` method?

**Comment by axelson** [comment]
Can we get common pitfalls such as this added to the README?

**Comment by kevinSuttle** [comment]
Yeah this is a really weird one. Causes very strange behavior in browsers.

**Comment by gilmoreorless** [comment]
Looks like this is fixed now that #23 has been merged and released in `1.2.0`
